packages/playground/src/router/index.ts

```diff
@@ -491,7 +509,7 @@ const router = createRouter({
               },
             },
             {
-              path: "freeflow",
+              path: ApplicationRoute.freeflow,
```

Comment:
i believe this route should be commented since the freeflow solution is currently hidden from the applications but with this present in the routes it can be accessed.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/1d439975-8ea1-455b-94bb-13ce4566032e)

Reply:
Yeah indeed but from where do you know that we have a freeflow solution? but yeah you are right I'll comment it out good catch.